R/ica.R

```diff
@@ -236,10 +253,11 @@ tidy.step_ica <- function(x, ...) {
     res$terms <- as.character(res$terms)
     res$component <- as.character(res$component)
     res <- as_tibble(res)
+    res$id <- x$id
   }
-  res$id <- x$id
```

Comment:
Actually can you just leave the addition of the id here? I think that would be best, that is actually what all the other steps do. Then you can remove all the lines above that deal with the id